NWEEI/NWEEI/Views/Article/Create.cshtml

```diff
@@ -1,57 +1,87 @@
-@model NWEEI.Models.Article
+@model NWEEI.ViewModels.CategorySelectorViewModel
```

Comment:
Interesting, so the View Model acts as a container for not only the article model but the category model as well.